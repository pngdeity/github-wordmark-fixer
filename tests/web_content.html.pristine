<!DOCTYPE html>
<html>
<head>
    <title>Test Page</title>
</head>
<body>
    <!-- This is a hidden comment about Github. -->
    <div style="display:none;">
        This is a hidden div with the word Github.
    </div>
    <p>
        Visible content about github.
    </p>
</body>
</html>
