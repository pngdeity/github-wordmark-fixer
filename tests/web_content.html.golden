<!DOCTYPE html>
<html>
<head>
    <title>Test Page</title>
</head>
<body>
    <!-- This is a hidden comment about GitHub. -->
    <div style="display:none;">
        This is a hidden div with the word GitHub.
    </div>
    <p>
        Visible content about GitHub.
    </p>
</body>
</html>
